ui/app.yaml

```diff
@@ -1,6 +1,4 @@
-runtime: python27
-api_version: 1
-threadsafe: true
+runtime: nodejs16
```

Comment:
I was having build failures that it turns out were unrelated, but I've been wanting to change this for a while since a Python runtime makes no sense.